index.ts

```diff
@@ -157,17 +157,27 @@ async function airdropOilTransaction(
     return [transaction, transaction.getTransferredAsset(0)];
 }
 
+async function fetchOilTracker(owner: string): Promise<H256> {
+    const utxoUrl = getConfig<string>("oil_utxo_url").toString();
```

Comment:
Should toString be called?

Reply:
No it doesn't have to. I tested and deleted it.